src/operator/tensor/elemwise_binary_op_basic.cc

```diff
@@ -89,12 +90,15 @@ NNVM_REGISTER_OP(_backward_mul)
   [](const NodeAttrs& attrs){
     return std::vector<std::pair<int, int> >{{0, 1}};
   })
-.set_attr<FCompute>("FCompute<cpu>", BinaryBackwardUseIn<cpu, mshadow_op::right,
-                                                              mshadow_op::left>);
-
-MXNET_OPERATOR_REGISTER_BINARY(_div)
-.add_alias("_Div")
-.set_attr<FCompute>("FCompute<cpu>", BinaryCompute<cpu, mshadow::op::div>)
+.set_attr<FInferStorageType>("FInferStorageType", ElemwiseStorageType<3, 2>)
+.set_attr<FCompute>("FCompute<cpu>", ElemwiseBinaryOp::BinaryBackwardUseIn<
+  cpu, mshadow_op::right, mshadow_op::left>)
+.set_attr<FComputeEx>("FComputeEx<cpu>", ElemwiseBinaryOp::BinaryBackwardUseInEx<
+  cpu, mshadow_op::right, mshadow_op::left>);;
+
+MXNET_OPERATOR_REGISTER_BINARY_LAUNCH_CPU_DR(elemwise_div, mshadow::op::div)
```

Comment:
it has to be, since 4/0 is nan or inf, or whatever it is that numpy returns for the same operation (it will return same result as numpy for th eunit tests). in my adagrad branch, it is possible to override this (and any sparse with dense result) with scatterred=True flag, although that naming may be revisited